internal/host/host.go

```diff
@@ -996,6 +997,48 @@ func (m *Manager) SetDiskSpeed(ctx context.Context, h *models.Host, path string,
 	return nil
 }
 
+func (m *Manager) resetDiskSpeedValidation(host *models.Host, log logrus.FieldLogger, db *gorm.DB) error {
+	bootDevice, err := hardware.GetBootDevice(log, m.hwValidator, host)
+	if err != nil {
+		return common.NewApiError(http.StatusInternalServerError, errors.New("Get boot device"))
+	}
+	var updatedHost models.Host
+	updatedHost.DisksInfo, err = common.ResetDiskSpeed(bootDevice, host.DisksInfo)
+	if err != nil {
+		return common.NewApiError(http.StatusInternalServerError, errors.New("Reset disk speed"))
+	}
+	return db.Model(&models.Host{}).Where("cluster_id = ? and id = ?", host.ClusterID.String(), host.ID.String()).Update(&updatedHost).Error
+}
+
+func (m *Manager) resetContainerImagesValidation(host *models.Host, db *gorm.DB) error {
+	return db.Model(&models.Host{}).Where("cluster_id = ? and id = ?", host.ClusterID.String(), host.ID.String()).Updates(
+		map[string]interface{}{
+			"images_status": "",
+		}).Error
+}
+
+func (m *Manager) ResetHostValidation(ctx context.Context, hostID, clusterID strfmt.UUID, validationID string, db *gorm.DB) error {
+	if db == nil {
+		db = m.db
+	}
+	var host models.Host
+	if err := db.Take(&host, "id = ? and cluster_id = ?", hostID.String(), clusterID.String()).Error; err != nil {
```

Comment:
i think better to use GetHostFromDB from common

Reply:
ok

Reply:
OK